extensions/landing-zone/version.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_version = ">= 1.3.0, <1.6.0"
+  required_providers {
+    # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
+    # OCP all inclusive requires 1.56.1, so breaking pin to lowest provider version. Note: PR tests include multiple-control-plans which is pinned
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">= 1.59.0"
```

Comment:
DA's should lock into an exact version. And we will use renovate to keep it up to date too. Please set this to 1.62.0 and create a renovate rule to bump it (like this -> https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/0f2626d4e2757a49dc97ea61780d15b44dfbb2de/renovate.json#L4-L15)